projects/demo/src/styles.scss

```diff
@@ -4,11 +4,11 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-@import '../../../node_modules/@cds/core/styles/module.reset.min.css';
-@import '../../../node_modules/@cds/core/styles/module.tokens.min.css';
-@import '../../../node_modules/@cds/core/styles/module.layout.min.css';
-@import '../../../node_modules/@cds/core/styles/module.typography.min.css';
-@import '../../../node_modules/@cds/core/styles/theme.dark.min.css';
+@import '../../../dist/@cds/core/styles/module.reset.min.css';
+@import '../../../dist/@cds/core/styles/module.tokens.min.css';
+@import '../../../dist/@cds/core/styles/module.layout.min.css';
+@import '../../../dist/@cds/core/styles/module.typography.min.css';
+@import '../../../dist/@cds/core/styles/theme.dark.min.css';
```

Comment:
How are consumer supposed to import these files? I think we need to build these files into the `@clr/ui` directory so that they are included in a published package.

Reply:
This will be handled once we make a choice and provide solution, as discussed in my reply to your first comment above.

Reply:
For now, changing the import to be through `@cds/core/styles/.....`, so we don't rely on direct relative paths and `dist`.